workflow-templates/im-test-cypress-schedule.yml

```diff
@@ -0,0 +1,130 @@
+# Workflow Code: SurprisedHedgehog_v22    DO NOT REMOVE
+# Purpose:
+#    Runs Cypress UI Health Check tests in Github Actions on a schedule.
+#    With the workflow you specify when to run this test.
+#
+# Frequency:
+#    - This workflow is intended to be reusable, and used once per repository.
+#
+# Projects to use this Template with:
+#    - Testing projects that use Cypress.
+#
+# TODO Prerequisites:
+#    - Add secret APPLICATIONINSIGHTS_CONNECTION_STRING to every environment you are testing.
+#    - Update your cypress test config to include the im.exporttoappinsights package. Instructions found here: https://github.com/im-practices/techhub/blob/main/docs/Azure/content/Health-Checks/Cypress-UI-Health-Check.md
+#    - If the project contains a local nuget.config remove it.  It interferes with restoring packages on the GitHub Action runners.
```

Comment:
Should this say npm instead of nuget.  Cypress doesn't use C# code.

Reply:
Updated to npm.